go/test/fuzzing/tablet_manager_fuzzer.go

```diff
@@ -33,7 +33,7 @@ func initTesting() {
 	testing.Init()
 }
 
-func FuzzTabletManager_ExecuteFetchAsDba(data []byte) int {
+func FuzzTabletManagerExecuteFetchAsDba(data []byte) int {
```

Comment:
Thankyou for fixing this, I saw this error in fuzzers too, but I had ignored it then!